db/views/hmis_households_v03.sql

```diff
@@ -0,0 +1,37 @@
+-- To update, run rails generate scenic:view hmis_households --replace
+
+SELECT
+  CONCAT("Enrollment"."HouseholdID", MAX("Project"."ProjectID"), MAX("Enrollment"."data_source_id")) AS id,
```

Comment:
Can you add in separators here?  There could be a weird situation where we've got like <50><11><2> and <501><1><2> which would turn into the same ID even though they're not. The format/length of each of these IDs is not guaranteed, since they could be imported from other systems.

Reply:
Fixed